HEC.FDA.Statistics/Distributions/Empirical.cs

```diff
@@ -69,13 +72,17 @@ public Empirical(double[] probabilities, double[] observationValues)
             BuildFromProperties();
             AddRules();
         }
+        /// <summary>
+        /// The probabilies and observation values must be in ascending order, and are assumed to be linked as coordinates. 
+        /// Counts should be equal. Min and max should be equal to the first and last values in the observationValues array.
+        /// </summary>
         public Empirical(double[] probabilities, double[] observationValues, double min, double max)
         {
             CumulativeProbabilities = probabilities;
             Quantiles = observationValues;
             Min = min;
             Max = max;
-            Truncated = true;
+            Truncated = true; //Not sure this is true. 
```

Comment:
I don't think this should be true by default. I think we should set this to False. 

This is a required Continous Distr property. 

Truncated gets used in InverseCDF to modify probability value